app/views/layouts/_timer.html.erb

```diff
@@ -4,7 +4,7 @@
     <i class="timer timer-text text-center"><%= t :time_left %></i>
     <i id="timer" class="timer timer-text text-center"></i>
     <script>
-      mumuki.startTimer('<%= end_time %>');
+      mumuki.startTimer('<%= end_time&.iso8601 %>');
```

Comment:
This one fixes the broken timer on exams. 
 ActiveSupport::TimeWithZone (Time as well) doesn't output valid standard date formats in to_s by default